index.js

```diff
@@ -143,7 +143,7 @@ function updateShrinkwrap(config) {
     }
 
     function rewritePath(key, object) {
-        if (object.resolved.search(/\.tgz|\.tar\.gz/) !== -1) {
+        if (object.resolved.search(/\.tgz|\.tar\.gz|\.git/) !== -1) {
```

Comment:
Although this is a quick fix, the better solution is to actually check if the tarball exists in the npm cache. Doing that would ensure that even if  "npm shrinkwrap" doesn't put in a "resolved" path that it will still be set (Currently having an issue where npm isn't setting the resolved path for some deps).